pkg/reconcile/pipeline/infinispan/handler/manage/conditions.go

```diff
@@ -247,8 +247,9 @@ func OperandStatus(i *ispnv1.Infinispan, phase ispnv1.OperandPhase, operand vers
 	}
 
 	return ispnv1.OperandStatus{
-		Image:   img,
-		Phase:   phase,
-		Version: operand.Ref(),
+		Deprecated: operand.Deprecated,
```

Comment:
I presume the status is refreshed on Operator respin or regularly?

Reply:
Good question, before it was only actually set for new Infinispan CRs. I have added a new check that will update the status if the existing value is not equal to that contained in the metadata.